packages/form-core/test-suites/ValidateMixinFeedbackPart.suite.js

```diff
@@ -517,5 +577,32 @@ export function runValidateMixinFeedbackPart() {
 
       expect(el._feedbackNode.feedbackData).to.deep.equal([]);
     });
+
+    describe('Global feedback messages', () => {
+      it('can set global validator messages', async () => {
+        Required.getMessage = () => `Fill in something`;
+        const el = await fixture(html`
+        <${tag}
+          .validators=${[new Required()]}
+        >${lightDom}</${tag}>
+      `);
+        await el.feedbackComplete;
+
+        expect(el._feedbackNode.feedbackData[0].message).to.equal('Fill in something');
+      });
+
+      it('can set global validator messages', async () => {
+        Required.getMessage = ({ article, fieldName }) => `Please enter ${article} ${fieldName}`;
+        ValidateMixin.fieldName = 'value';
```

Comment:
ValidateMixin is a function, not a constructor. This should be considered pseudo code